theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/CurveEditorPopover.tsx

```diff
@@ -233,225 +183,128 @@ const CurveEditorPopover: React.FC<
-                      width="18"
-                      height="18"
-                      viewBox={`0 0 ${1 + svgPadding * 2} ${
-                        1 + svgPadding * 2
-                      }`}
-                      fill="none"
-                      xmlns="http://www.w3.org/2000/svg"
-                    >
-                      <path
-                        d={`M${svgPadding} ${1 + svgPadding} C${
-                          easing[0] + svgPadding
-                        } ${1 - easing[1] + svgPadding} ${
-                          easing[2] + svgPadding
-                        } ${1 - easing[3] + svgPadding} ${
-                          1 + svgPadding
-                        } ${svgPadding}`}
-                        stroke={svgColor}
-                        strokeWidth="0.08"
-                      />
-                      <circle
-                        cx={svgPadding}
-                        cy={1 + svgPadding}
-                        r={svgCircleRadius}
-                        fill={svgColor}
-                      />
-                      <circle
-                        cx={1 + svgPadding}
-                        cy={svgPadding}
-                        r={svgCircleRadius}
-                        fill={svgColor}
-                      />
-                    </svg>
-                  </EasingCurveContainer>
-                  <span>
-                    {useQuery ? (
-                      <span
-                        dangerouslySetInnerHTML={{
-                          __html: fuzzySort.highlight(
-                            presetSearchResults[index] as any,
-                          )!,
-                        }}
-                      />
-                    ) : (
-                      preset.label
-                    )}
-                  </span>
-                </EasingOption>
-              )
-            })}
-          </div>
-        </OptionsContainer>
-      )}
-    </Container>
+    <Grid>
+      <SearchBox
+        value={filter}
+        placeholder="Search presets..."
+        onChange={(e) => {
+          r.current = true
+          setFilter(e.target.value)
+
+          const a = bezierPointsFromString(filter)
+          if (a) fns.permanentlySetValue(e.target.value)
+        }}
+        ref={inputRef}
+        onKeyDown={onSearchKeyDown}
+      />
+      <OptionsContainer>
+        {displayedPresets.map((preset, index) => (
+          <EasingOption
+            key={preset.label}
+            easing={preset}
+            tabIndex={0}
+            onKeyDown={onEasingOptionKeydown(preset)}
+            ref={optionsRef.current[preset.label]}
+            onClick={() => {
+              fns.permanentlySetValue(preset.value)
+              //props.onRequestClose()
+            }}
+            // Temporarily apply on hover
+            onMouseOver={() => {
+              // When previewing with hover, we don't want to set the filter too
+              //fns.temporarilySetValue(preset.value)
```

Comment:
Change: This comment was previously attached to an uncommented following line. But now the following line is commented so it has a changed meaning.

If you're not sure about removing this dead code, and want to keep it, then at least make sure the comment look like:
```ts
// // When previewing with hover, we don't want to set the filter too
// fns.temporarilySetValue(preset.value)
```

So it is obvious that the entire code with the comment is commented out.

Alternatively, comment out the entire `onMouseOver` value with a comment describing why it might be interesting to keep the code around.

Perhaps just delete the code here entirely and capture any uncertainty about this in a different nearby comment capturing the thought process / development journey.